whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapExceptionMapper.java

```diff
@@ -69,12 +68,12 @@ public Response toResponse(final Exception exception) {
         }
 
         if (exception instanceof EmptyResultDataAccessException) {
-            return Response.status(HttpServletResponse.SC_NOT_FOUND).entity(createErrorEntity(HttpServletResponse.SC_NOT_FOUND, "not found")).build();
+            return Response.status(HttpServletResponse.SC_NOT_FOUND).entity(createErrorEntity(HttpServletResponse.SC_NOT_FOUND, "404 Not Found")).build();
```

Comment:
RdapQueryHandler should catch this and later throw as rdapException

Reply:
Actually, all the places related to Rdap which are catching this error are doing a continue; so this should not being here, we are not using it

Reply:
Can you remove it then?  I see we are sending back 404 in case we get an empty result.